{"_id":"91d99160-f7e0-33f1-b51a-873f1fff3ecf","authorName":"Zarat","authorEmailEncrypted":"niCJHBfqzWeZyXHs0fEis4tyPa4E+n5yWL6MeRbcERB5SlKu99bd9rbbKo4pVbHLMYRs7S5a1ePacQ/Qp5GqzdqgwNy6/2o36qqNucyhZr1QChH011Dz2CrZq63WUvjm80zINR3YOVOGXRbCc58ixQxDNmjeozIc2byeq4DZna7KA3dDx2J85z8q78HsvgD4bUIZ5KxIixsV6JE6g9gEq6+ojd/92IItnmUZOx3O5+JmqaG3dUTm0VNxb4tDhA9CFwicj4PwHDPxsDbzUjvNxhFLn8zSb3DmtTa7he1VMOikvJF8CQ5Vcjwu78Zo58n54+QkU4XM8Rcf0mbRxLpkvZhPCQaYiyGfRi2mx54c/Sj2SppGiHIEXenBzNr/XASZ++wBLPiEvicNZVWNJjnFnU8dDC5Bea0qRZjauJP/riYDZySNkLvufHIkoSholuQt5zh60AF+nlE7aS5dQtkJ0F0A+QsN1LxrA0MPOhbYEqij64LujBR83DlBTy1xVCY6q9tesYYrGu8cisIfasvPT7B3nAr2A1MNdf9UQbmQGB3BDn33Gu6KTHHfY3vw5Uk/8fyOVD7Oyx4knwXwLk3CcbEQ07CUnQ6BiNVBSbV3jazKInZB4m8V4nC6ay3urb3RVJYEQvH1U0PcUok7R+45GD6V/+BXHj1Y6mZ51lEcCTA=","authorEmailMD5":"d307e42a92fbce947bfa9fae4400e95f","message":"If you're on 4.5 you can use System.Runtime.ExceptionServices.ExceptionDispatchInfo to store the exception and rethrow it without overwriting the stack trace.","postId":"2014_06_await-in-catch-and-finally-196df97f-fc37-3a25-b9e6-1b95025b3836","replyTo":"e0dc4f0e-ae83-39b8-9ce0-b23ee457470f","date":"2014-06-16T08:23:07Z","timestamp":1402906987,"authorUserId":"disqus:disqus_bCQ4ScqAv7"}